draft-ietf-rats-corim.md

```diff
@@ -387,8 +387,18 @@ The following describes each child element of this type.
 
 ### Profile Types {#sec-corim-profile-types}
 
-A profile specifies which of the optional parts of a CoRIM are required, which
-are prohibited and which extension points are exercised and how.
+Profiling is the mechanism that allows the base CoRIM schema to be customised to fit a specific Attester or Attesters' family.
```

Comment:
not sure 'Attester's family' is a concrete concept. It may generate more questions.

Reply:
dropped in [cee17f1](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/76/commits/cee17f19773f8503d87dc204bc00a0f924acfe75)